webapp/components/ContributionForm/ContributionForm.spec.js

```diff
@@ -293,5 +297,88 @@ describe('ContributionForm.vue', () => {
         })
       })
     })
+
+    describe('Events', () => {
+      beforeEach(() => {
+        propsData.createEvent = true
+        wrapper = Wrapper()
+      })
+
+      it('has event data block', () => {
+        expect(wrapper.find('div.eventDatas').exists()).toBe(true)
+      })
+
+      describe('is online event', () => {
+        it('has false as default', () => {
+          expect(wrapper.vm.formData.eventIsOnline).toBe(false)
+        })
+
+        it('has input for event location', () => {
+          expect(wrapper.find('input[name="eventLocationName"]').exists()).toBe(true)
+        })
+
+        describe('click is online event', () => {
+          beforeEach(() => {
+            wrapper.find('input[name="eventIsOnline"]').setChecked(true)
+          })
+
+          it('has no input for event location', () => {
+            expect(wrapper.find('input[name="eventLocationName"]').exists()).toBe(false)
+          })
+        })
+
+        describe('invalid form', () => {
```

Comment:
it is unclear from the test description why this birthday is illegal

Reply:
Because it was during the pandemic. It is not valid because all the event data is missing